.github/workflows/containers-and-az-pool.yaml

```diff
@@ -1,5 +1,4 @@
-name: Test Model Image Build
-run-name: (2_pre) Test Model Image - "${{ github.event.head_commit.message }}"
```

Comment:
Do we want fix these references rather than dropping? What are the pros/cons?

Reply:
As far as I understand, the run name is either the commit message (the default) or something else that cannot hold the commit message (AFAIK). With the default, all runs within a single commit (push) will have the same name.